plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -765,9 +766,22 @@ public List<String> getFileStatusCacheTables()
     }
 
     @Config("hive.file-status-cache-tables")
-    public HiveConfig setFileStatusCacheTables(List<String> fileStatusCacheTables)
+    public HiveConfig setFileStatusCacheTables(List<String> tables)
     {
-        this.fileStatusCacheTables = ImmutableList.copyOf(fileStatusCacheTables);
+        this.fileStatusCacheTables = ImmutableList.copyOf(tables);
+        return this;
+    }
+
+    public List<String> getFileStatusCacheExcludedTables()
+    {
+        return fileStatusCacheExcludedTables;
+    }
+
+    @Config("hive.file-status-cache-excluded-tables")
```

Comment:
How about `hive.file-status-cache-tables.excluded`

Reply:
I would actually go with `hive.file-status-cache.excluded-tables`, as in "the list of excluded tables for the file status cache in the hive connector". It would sit side-by-side with other properties of the file status cache, such as `hive.file-status-cache.max-retained-size`

Reply:
I'll stick with what we have then, unless someone feels strongly.  Thank you